packages/flutter/lib/src/widgets/scroll_view.dart

```diff
@@ -310,9 +321,25 @@ abstract class ScrollView extends StatelessWidget {
         return buildViewport(context, offset, axisDirection, slivers);
       },
     );
-    return primary && scrollController != null
-      ? PrimaryScrollController.none(child: scrollable)
-      : scrollable;
+    final Widget scrollableResult = primary && scrollController != null
+        ? PrimaryScrollController.none(child: scrollable)
+        : scrollable;
+
+    if (keyboardDismissBehavior == ScrollViewKeyboardDismissBehavior.onDrag) {
+      final FocusNode dismissNode = FocusNode();
+      return NotificationListener<ScrollUpdateNotification>(
+        child: scrollableResult,
+        onNotification: (ScrollUpdateNotification notification) {
+          final FocusScopeNode focusScope = FocusScope.of(context);
+          if (notification.dragDetails != null && focusScope.focusedChild != dismissNode) {
+            focusScope.requestFocus(dismissNode);
+          }
+          return true;
```

Comment:
We probably shouldn't return true here -eating scroll notifications this low in the tree is likely to break people further up who also want to know about scrolling.